keyboards/crkbd/keymaps/hvp/keymap.c

```diff
@@ -16,6 +16,7 @@ extern uint8_t is_master;
 #define _LOWER 1
 #define _RAISE 2
 #define _ADJUST 3
+#define D_NAVI MT(MOD_LCTL | MOD_LSFT, KC_D)
```

Comment:
You're using this in multiple locations.  You may want to move this to your `hvp.h` file, instead, so it's only defined once

Reply:
Moved it to the hvp.h file and tested ok